data-guides/intro.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
This stuff does not make so much sense now we have moved out of patterns. The general move of stuff out of patterns is an example of a major structural change we *should* have discussed in a github issue first before implementing.